TransportShop.DAL/EF/DataContext.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Microsoft.EntityFrameworkCore;
+using TransportShop.DAL.EF.EntitiesConfigurations;
+using TransportShop.DAL.Entities;
+
+namespace TransportShop.DAL.EF
+{
+    public class DataContext : DbContext
+    {
+        public DbSet<Product> Products { get; set; }
+        public DbSet<Category> Categories { get; set; }
+        public DbSet<Order> Orders { get; set; }
+        public DbSet<OrderItem> OrderItems { get; set; }
+        public DbSet<User> Users { get; set; }
+
+        protected override void OnModelCreating(ModelBuilder modelBuilder)
+        {
+            modelBuilder.ApplyConfiguration(new ProductConfiguration());
+            modelBuilder.ApplyConfiguration(new CategoryConfiguration());
+            modelBuilder.ApplyConfiguration(new OrderConfiguration());
+            modelBuilder.ApplyConfiguration(new OrderItemConfiguration());
+            modelBuilder.ApplyConfiguration(new UserConfiguration());
```

Comment:
you can use modelBuilder.ApplyConfigurationsFromAssembly(Assembly.GetExecutingAssembly());